pkg/apimachinery/identity/wrapper.go

```diff
@@ -0,0 +1,120 @@
+package identity
+
+import (
+	"time"
+
+	"github.com/grafana/authlib/claims"
+)
+
+var _ claims.IdentityClaims = &IDClaimsWrapper{}
+var _ claims.AccessClaims = &IDClaimsWrapper{}
+
+type IDClaimsWrapper struct {
+	Source Requester
+}
+
+// GetAuthenticatedBy implements claims.IdentityClaims.
+func (i *IDClaimsWrapper) AuthenticatedBy() string {
+	return i.Source.GetAuthenticatedBy()
+}
+
+// GetDisplayName implements claims.IdentityClaims.
+func (i *IDClaimsWrapper) DisplayName() string {
+	return i.Source.GetDisplayName()
+}
+
+// GetEmail implements claims.IdentityClaims.
+func (i *IDClaimsWrapper) Email() string {
+	return i.Source.GetEmail()
+}
+
+// GetEmailVerified implements claims.IdentityClaims.
+func (i *IDClaimsWrapper) EmailVerified() bool {
+	return i.Source.IsEmailVerified()
+}
+
+// GetIdentityType implements claims.IdentityClaims.
+func (i *IDClaimsWrapper) IdentityType() claims.IdentityType {
+	return claims.IdentityType(i.Source.GetIdentityType())
```

Comment:
@kalleep -- after you merge your PR, we can replace the identity type and remove this cast

Reply:
I decided to split up my pr in several smaller ones so I think we can go ahead and replace in this one if you want. Another thing we could do as a first step is to re-export them all from this package WDYT?